lib/acl.c

```diff
@@ -136,11 +136,13 @@ process_acl(struct gatekeeper_if *iface, unsigned int lcore_id,
 void
 destroy_acls(struct acl_state *astate)
 {
-	unsigned int numa_nodes = get_net_conf()->numa_nodes;
+	struct net_config *net_conf = get_net_conf();
```

Comment:
This change made sense before the previous review, but now it does not since it's just looking up the field `net_conf->numa_nodes` at the end of the `for` loop instead of using variable `numa_nodes`.

This comment applies to a number of changes in this file, namely `lib/acl.c`.